frontend/website-redesign/src/components/Icon.re

```diff
@@ -39,52 +41,943 @@ type kind =
+         <rect x="9" y="6" width="1" height="1" fill="#2D2D2D" />
+         <rect x="10" y="5" width="1" height="1" fill="#2D2D2D" />
+       </svg>
+     | ArrowLeftLarge =>
+       <svg>
+         <rect
+           width="24"
+           height="1"
+           transform="matrix(-1 0 0 1 24 11)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 2 12)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 2 10)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 3 9)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 4 8)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 5 7)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 6 6)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 7 5)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 8 4)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 9 3)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 10 2)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 3 13)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 4 14)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 5 15)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 6 16)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 7 17)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 8 18)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 9 19)"
+           fill="#2D2D2D"
+         />
+         <rect
+           width="1"
+           height="1"
+           transform="matrix(-1 0 0 1 10 20)"
+           fill="#2D2D2D"
+         />
+       </svg>
+
```

Comment:
looks great! just a minor nitpick, can we get consistent spacing between all variants in the file?